tests/cephadm/test_cephadm_ansible_wrapper.py

```diff
@@ -109,6 +109,12 @@ def run(ceph_cluster, **kwargs):
     module = aw.get("module")
     module_args = aw.get("module_args", {})
 
+    # Check if the scenario includes docker
+    if module_args.get("docker"):
```

Comment:
This module arg is specific to bootstrap module. Better to move under `cephadm_bootstrap` check

Reply:
Its not only for bootstrap Vaibhav, the other scenario for logging into the registry via docker is also there. So it cannot be clubbed under bootstrap alone

Reply:
ok